language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/util/rename/RenameUtil.java

```diff
@@ -125,25 +128,68 @@ private RenameUtil() {
         if (onImportDeclarationNode(context, nodeAtCursor)) {
             return Collections.emptyMap();
         }
+
         if (QNameReferenceUtil.onModulePrefix(context, nodeAtCursor)) {
-            return handleQNameReferenceRename(context, document.get(), nodeAtCursor, newName);
+            return handleQNameReferenceRename(context, document.get(), nodeAtCursor, newName, honorsChangeAnnotations);
```

Comment:
Refer the previous suggestion regarding the `honorsChangeAnnotations` flag related API